src/win32/git2.rc

```diff
@@ -9,7 +9,18 @@
 # define LIBGIT2_COMMENTS "For more information visit http://libgit2.github.com/"
 #endif
 
+#ifdef GCC_WINDRES
+# define STRINGIZE(x) #x
+# define STRINGIZE_(x) STRINGIZE(x)
+#else
+# define STRINGIZE_(x) x
```

Comment:
It doesn't make much difference in the grand scheme of things, but if you're making changes anyway, things with spurious underscores tend to be what we use to denote _internal_ components.  So we would normally reverse this, and do something like:

```
#define _STR(x) #x
#define STR(x) _STR(x)
```

and then use the name _without_ the underscore.

Reply:
Addressed!